test/mpi-cpp-fortran/base/CMakeLists.txt

```
@@ -0,0 +1,33 @@
+cmake_minimum_required(VERSION 3.14)
+
+project(mpi-cpp-fortran LANGUAGES CXX)
+
+set(BLT_CXX_STD c++14 CACHE STRING "")
+set(ENABLE_MPI On CACHE BOOL "")
+
+# Load BLT
+if (NOT BLT_LOADED)
+  include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
+endif()
```

```suggestion
include(${BLT_SOURCE_DIR}/SetupBLT.cmake)
```

This is legacy from an old BLT but it guards against this itself inside `SetupBLT.cmake`